packages/blade/src/components/FileUpload/FileUpload.web.tsx

```diff
@@ -158,6 +159,7 @@ const _FileUpload: React.ForwardRefRenderFunction<BladeElementRef, FileUploadPro
     if (!hasValidationErrors) {
       handleFilesChange(droppedFiles);
       onDrop?.({ name, fileList: allFiles });
+      fireNativeEvent(inputRef, ['change', 'input']);
```

Comment:
Check the normal DOM inputs if we actually fire both of these or not. I think they do but still confirm once.

Reply:
checked for ```  <input type="file" /> ``` they do fire both change and  input .